### PR TITLE
Updated app name

### PR DESCRIPTION
**Pull request by amit-metri**
-updated app name

### CODE REVIEW

File: common/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="app_name">My Health BC</string>
+    <string name="app_name">Health Gateway</string>
     <string name="notification_channel_name">My Health</string>
```
Comment (pinakinkansara) [inline common/src/main/res/values/strings.xml:4]: Change this to Health Gateway

File: common/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
     <string name="notification_channel_work">My Health Notifications</string>
```
Comment (pinakinkansara) [inline common/src/main/res/values/strings.xml:5]: Change to Health Gateway

**Review by pinakinkansara — CHANGES_REQUESTED** [review]

File: common/src/main/res/values/strings.xml (hunk 1 above)
Comment (amit-metri) [reply common/src/main/res/values/strings.xml]: Done

File: common/src/main/res/values/strings.xml (hunk 2 above)
Comment (amit-metri) [reply common/src/main/res/values/strings.xml]: Done

**Review by pinakinkansara — APPROVED** [review]

**Pull request merged**